src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/CompilerServices/FunctionPointerOps.cs

```diff
@@ -17,42 +17,7 @@ public static class FunctionPointerOps
         private const int FatFunctionPointerOffset = 2;
 #endif
 
-        private struct GenericMethodDescriptorInfo : IEquatable<GenericMethodDescriptorInfo>
```

Comment:
The overhead of using tuples instead of custom struct:
- `ToString` implementation (pure overhead)
- `String.Concat` (surprisingly not used in hello world, very likely to be used otherwhere)
- Larger implementation of `GetHashCode`, involving `HashCode.Combine`. Will be eliminated once `HashCode.Combine` is used elsewhere.

Reply:
The conventional wisdom is that using Tuples in low-level code is bad for trimming. Have you verified that it is a net saving to use Tuples in this change?

Reply:
It's net (significant) saving to fold multiple different structs into the same tuple, and net (little) regression to replace one struct with one tuple. The most-saving option should be creating sharable tuple-like struct, however it will be a regression again if user uses the same tuple.